connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConnector.java

```diff
@@ -162,7 +164,9 @@ public void start(Map<String, String> props) {
         sourceAdminClient = config.forwardingAdmin(config.sourceAdminConfig("replication-source-admin"));
         targetAdminClient = config.forwardingAdmin(config.targetAdminConfig("replication-target-admin"));
         useIncrementalAlterConfigs =  !config.useIncrementalAlterConfigs().equals(MirrorSourceConfig.NEVER_USE_INCREMENTAL_ALTER_CONFIGS);
-        offsetSyncsAdminClient = config.forwardingAdmin(config.offsetSyncsTopicAdminConfig());
+        if (config.emitOffsetSyncEnabled()) {
```

Comment:
This client is only used to create the topic in `createOffsetSyncsTopic()`. I wonder if we can only create it `createOffsetSyncsTopic()`, create the topic and directly close it instead of having it as a field. WDYT?